daisy/step_create_instances.go

```diff
@@ -55,11 +56,18 @@ Loop:
 					}
 				}
 				if numErr > 10 {
-					w.LogStepInfo(s.name, "CreateInstances", "Instance %q: error getting serial port: %v", i.Name, err)
+					// Quit silently if we were able to read *some* data from the instance,
+					// since there's a race condition where an instance can shut down
+					// fast enough that the call to InstanceStatus will return a 404.
```

Comment:
Why don't we just add a check for sErr, if we get a 404 then quit the same as if we got "TERMINATED", "STOPPED", "STOPPING"

Reply:
I've seen 404s returned prior to PROVISIONING, so we'd still need to decide whether the 404 meant to quit or to wait longer.

https://cloud.google.com/compute/docs/instances/instance-life-cycle

Reply:
Ack